### ISSUE TITLE
Crashes right away on IE Edge

### ISSUE THREAD

**Issue by bbenezech** [muaz-khan/DetectRTC#15]
SCRIPT438: Object doesn't support property or method 'indexOf'

``` javascript
        // trim the fullVersion string at semicolon/space if present
        if ((ix = fullVersion.indexOf(';')) !== -1) {
            fullVersion = fullVersion.substring(0, ix);
        }

```

**Comment by bbenezech** [comment]
Looks like fullVersion is an integer: 10240 in my case.

You can reproduce easily on https://www.webrtc-experiment.com/DetectRTC/

**Comment by brianpursley** [comment]
Should be resolved now after #16
